cpp/modmesh/toggle/RadixTree.hpp

```diff
@@ -225,17 +225,16 @@ class CallProfiler
     ~CallProfiler() = default;
 
     // Called when a function starts
-    void start_caller(const std::string & caller_name, std::function<void()> cancel_callback)
```

Comment:
remove unused arg

Reply:
Please don't mark conversation as resolved.  If you want to say it is resolved, leave a comment saying so.

Marking conversation as resolved makes it hard to keep track of discussion history.